ui/index.d.ts

```diff
@@ -1371,7 +1371,7 @@ export declare abstract class AzExtTreeFileSystem<TItem extends AzExtTreeItem> i
 
     public constructor(tree: AzExtTreeDataProvider);
 
-    public get onDidChangeFile(): Event<FileChangeEvent[]>;
+    public readonly onDidChangeFile: Event<FileChangeEvent[]>;
```

Comment:
As you mentioned, I didn't get any errors with using get in our index.d.ts.  Could you change this back?